nf_core/pipelines/lint/rocrate_readme_sync.py

```diff
@@ -0,0 +1,59 @@
+import json
+import logging
+from pathlib import Path
+
+log = logging.getLogger(__name__)
+
+
+def rocrate_readme_sync(self):
+    """
+    Check if the RO-Crate description in ro-crate-metadata.json matches the README.md content.
+    If the --fix is set, the RO-Crate description will be updated to match the README.md content.
+    """
+
+    warned = []
+    passed = []
+    failed = []
+
+    # Check if the file exists before trying to load it
+    metadata_file = Path(self.wf_path, "ro-crate-metadata.json")
+    readme_file = Path(self.wf_path, "README.md")
+
+    # Only proceed if both files exist
+    if not (metadata_file.exists() and readme_file.exists()):
+        if not metadata_file.exists():
+            failed.append("ro-crate-metadata.json not found")
```

Comment:
If the file doesn't exist, we can ignore the test
```suggestion
            ignored.append("`ro-crate-metadata.json` not found")
```